app/src/main/res/values/autocomplete_words.xml

```diff
@@ -767,4 +767,372 @@
+        <item>UPDATETEXT</item>
+        <item>USE</item>
+        <item>USER</item>
+        <item>USING</item>
+        <item>VALUES</item>
+        <item>VARYING</item>
+        <item>VIEW</item>
+        <item>WAITFOR</item>
+        <item>WHEN</item>
+        <item>WHERE</item>
+        <item>WHILE</item>
+        <item>WITH</item>
+        <item>WRITETEXT</item>
+        <item>add</item>
+        <item>all</item>
+        <item>alter</item>
+        <item>and</item>
+        <item>any</item>
+        <item>apply</item>
+        <item>as</item>
+        <item>asc</item>
+        <item>authorization</item>
+        <item>backup</item>
+        <item>begin</item>
+        <item>between</item>
+        <item>break</item>
+        <item>browse</item>
+        <item>bulk</item>
+        <item>by</item>
+        <item>cascade</item>
+        <item>case</item>
+        <item>check</item>
+        <item>checkpoint</item>
+        <item>close</item>
+        <item>clustered</item>
+        <item>coalesce</item>
+        <item>collate</item>
+        <item>column</item>
+        <item>commit</item>
+        <item>compute</item>
+        <item>connect</item>
+        <item>constraint</item>
+        <item>contains</item>
+        <item>containstable</item>
+        <item>continue</item>
+        <item>convert</item>
+        <item>create</item>
+        <item>cross</item>
+        <item>current</item>
+        <item>current_date</item>
+        <item>current_time</item>
+        <item>current_timestamp</item>
+        <item>current_user</item>
+        <item>cursor</item>
+        <item>database</item>
+        <item>dbcc</item>
+        <item>deallocate</item>
+        <item>declare</item>
+        <item>default</item>
+        <item>delete</item>
+        <item>deny</item>
+        <item>desc</item>
+        <item>disk</item>
+        <item>distinct</item>
+        <item>distributed</item>
+        <item>double</item>
+        <item>drop</item>
+        <item>dummy</item>
+        <item>dump</item>
+        <item>else</item>
+        <item>end</item>
+        <item>errlvl</item>
+        <item>escape</item>
+        <item>except</item>
+        <item>exec</item>
+        <item>execute</item>
+        <item>exists</item>
+        <item>exit</item>
+        <item>fetch</item>
+        <item>file</item>
+        <item>fillfactor</item>
+        <item>following</item>
+        <item>for</item>
+        <item>foreign</item>
+        <item>freetext</item>
+        <item>freetexttable</item>
+        <item>from</item>
+        <item>full</item>
+        <item>function</item>
+        <item>goto</item>
+        <item>grant</item>
+        <item>group</item>
+        <item>having</item>
+        <item>holdlock</item>
+        <item>identity</item>
+        <item>identitycol</item>
+        <item>identity_insert</item>
+        <item>if</item>
+        <item>in</item>
+        <item>index</item>
+        <item>inner</item>
+        <item>insert</item>
```

Comment:
can we suggest `INSERT` instead of `insert`, when user has typed `in`?

Reply:
We can but then we have to change autocomplete logic.

Reply:
Ok, for this release lets remain this logic, but in future releases we should think about it [Apps-1674](https://vyahhi.myjetbrains.com/youtrack/issue/APPS-1674)